### PR TITLE
check vdsm is not installed only during host deploy

### PR DESCRIPTION
**Pull request by dangel101**
Set correct virt DNF module version during host upgrade flow to aling with RHEL version that host is being upgraded to.

Bug-Url: https://bugzilla.redhat.com/2055136

### CODE REVIEW

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/av-setup.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@
     - host_deploy_origin_type != ovirt_origin_type
     - el_ver|int >= 8
     - not node_host|bool
-    - not vdsm_preinstalled|bool
+    - host_action is not defined or (host_action is defined and host_action == 'host_deploy' and not vdsm_preinstalled|bool)
```
Comment (mwperina) [inline packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/av-setup.yml:38]: Can't it be simplified?
 - host_action is not defined or (host_action == 'host_deploy' and not vdsm_preinstalled|bool)```
```

**Review by mwperina — CHANGES_REQUESTED** [review]

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/av-setup.yml (hunk 1 above)
Comment (dangel101) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/av-setup.yml]: done

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/av-setup.yml [hunk 2]
```diff
@@ -35,4 +35,4 @@
     - host_deploy_origin_type != ovirt_origin_type
     - el_ver|int >= 8
     - not node_host|bool
-    - host_action is not defined or (host_action == 'host_deploy' and not vdsm_preinstalled|bool)
+    - host_action is defined and host_action == 'host_deploy' and not vdsm_preinstalled|bool # use for host deploy only
```
Comment (mwperina) [inline packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-facts/tasks/av-setup.yml:38]: Wouldn't bellow be more readable:

```
when:
    - host_deploy_origin_type != ovirt_origin_type
    - el_ver|int >= 8
    - not node_host|bool
     # Below condittions limitts he execution for host deploy only
    - host_action is defined
    - host_action == 'host_deploy'
    - not vdsm_preinstalled|bool
```

**Review by mwperina — APPROVED** [review]
Looks good, please add a verification finished comment, so we can merge afterwards

**Comment by dangel101** [comment]
verified upgrading from 8.5 to 8.6

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**